app/views/contacts/_fields.slim

```diff
@@ -1,5 +1,8 @@
 .row
-  .col-xs-12.col-md-6= f.input :name
+
+  - unless defined? @department
+    .col-xs-12.col-md-6= f.input :first_name
+  .col-xs-12.col-md-6= f.input :last_name, label: !defined?(@department) ? t('.last_name') : t('.name')
```

Comment:
just keep the name field for the departments, will make life so much easier everywhere.